packages/component/src/Composer.js

```diff
@@ -162,12 +163,34 @@ const Composer = ({
     return sendTyping;
   }, [sendTyping, sendTypingIndicator]);
 
+  const patchedStyleOptions = useMemo(() => {
+    const patchedStyleOptions = { ...styleOptions };
+
+    if (typeof groupTimestamp !== 'undefined' && typeof patchedStyleOptions.groupTimestamp === 'undefined') {
+      console.warn(
+        'Web Chat: "groupTimestamp" has been moved to "styleOptions". This deprecation migration will be removed on or after January 1 2022.'
```

Comment:
isn't the standart 1 year?

Reply:
AFAIK, standard is 2 years and 2 versions.